v1/pipeline/steps/cd/k8s-rolling-step-info.yaml

```diff
@@ -3,56 +3,20 @@ allOf:
 - $ref: ../../common/step-spec-type.yaml
 - type: object
   properties:
-    commandFlags:
+    command_flags:
       oneOf:
       - type: array
         items:
           $ref: k8s-step-command-flag.yaml
       - type: string
-        pattern: ^<\+input>((\.)((executionInput\(\))|(allowedValues|default|regex)\(.+?\)))*$
-        minLength: 1
-    delegateSelectors:
+    delegates:
+      $ref: ../../common/delegate.yaml
+    prune:
       oneOf:
-      - type: array
-        items:
-          type: string
-      - type: string
-        pattern: (<\+.+>.*)
-        minLength: 1
-    pruningEnabled:
+        - type: boolean
+        - type: string
+    dry_run:
```

Comment:
are we not following default value true rule?
if we are following this would be against that.

Reply:
I think the rule is that make the fields optional and adopt the default behaviour for the execution. So `skip_dry_run` is false by default in V0. If we call it dry_run in V1 then the default behaviour will be as if the field is true until provided false or we can call it `skip_dry_run` only in V1 also. @archit-harness what do you think we should do here?

Reply:
Updated the filed name to skip_dry_run so that the default value would remain as false.

Reply:
+1